esp/scm/ws_workunits.ecm

```diff
@@ -1471,9 +1471,11 @@ ESPresponse [exceptions_inline] WUQueryDetailsResponse
     [min_ver("1.46")] string CompileTime;
     [min_ver("1.46")] ESParray<string> LibrariesUsed;
     [min_ver("1.46")] int CountGraphs;
-    [min_ver("1.46")] ESParray<string> GraphIds;
+    [min_ver("1.46"), depr_ver("1.62")] ESParray<string> GraphIds;
```

Comment:
Does EclWatch code need to change (does it use default version of these interfaces?)

@GordonSmith